CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 # Upcoming
 
 * Empty functions `function () {}` replaced by `angular.noop`
+* Fixed comment examples reference old service names
```

Comment:
should this line read 
`Fixed comment examples referencing old service names` ?